db/migrate/20201123124006_create_sdg_local_targets.rb

```diff
@@ -0,0 +1,21 @@
+class CreateSDGLocalTargets < ActiveRecord::Migration[5.2]
```

Comment:
How about adding some local targets to `dev_seeds` so we can test how this section will look in a real world usage?

Reply:
Totally agree!